doc/newsfragments/1831_new.driver_log_capture.rst

```diff
@@ -0,0 +1,2 @@
+Added :py:meth:`_record_resource_exception <testplan.common.entity.base.Environment._record_resource_exception>` to fetch error log messages from failed resources during Environment start or stop.
```

Comment:
This is too technical. No need to mention the private helper, that is an implementation details.

Reply:
Alright, noted.